src/main/java/services/gorest/actions/comment/CreateComment.java

```diff
@@ -0,0 +1,34 @@
+package services.gorest.actions.comment;
+
+import io.restassured.response.Response;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+import services.gorest.actions.GoRestActions;
+import services.gorest.models.Comment;
+import utils.reusable.specifications.ReusableSpecifications;
+
+public class CreateComment extends GoRestActions {
+
+    private String CREATE_COMMENT_URL = getBaseUri() + COMMENTS_ENDPOINT;
+
+
+    @Step("I add a new comment")
+    public Response createNewComment(Comment comment) {
+        Response response = SerenityRest.rest().given().log().all()
+                .spec(ReusableSpecifications.authorizedRequestSpec())
+                .baseUri(CREATE_COMMENT_URL)
+                .when()
+                .body(comment)
+                .post();
+        response.then().log().all().spec(ReusableSpecifications.responseSpec());
```

Comment:
Please avoid doing the same things we already reviewed it. Remove the response check